noetic/distribution.yaml

```diff
@@ -3,8 +3,6 @@
 # see REP 143: http://ros.org/reps/rep-0143.html
 ---
 release_platforms:
-  debian:
-  - buster
```

Comment:
This change governs the release platforms that Bloom uses when generating release data for Noetic. I didn't expect to see it here based on the PR title and description.

In my [little checklist](https://github.com/ros-infrastructure/ros_buildfarm_config/pull/221#issuecomment-1444599141), this change happens much later than the removal of the repo checker config. As a result, I think this PR was merged somewhat prematurely.

Reply:
Reverted in https://github.com/ros/rosdistro/pull/36303 and redone in https://github.com/ros/rosdistro/pull/36387